incubator/hnc/pkg/reconcilers/anchor_test.go

```diff
@@ -75,10 +75,13 @@ var _ = Describe("Anchor", func() {
 
 		// Change the bar's parent. Additionally change another field to reflect the
 		// update of the HC instance (hc.Spec.AllowCascadingDelete).
-		barHier := getHierarchy(ctx, barName)
-		barHier.Spec.Parent = "other"
-		barHier.Spec.AllowCascadingDelete = true
-		updateHierarchy(ctx, barHier)
+		Eventually(func() bool {
+			barHier := getHierarchy(ctx, barName)
+			barHier.Spec.Parent = "other"
+			barHier.Spec.AllowCascadingDelete = true
+			updateHierarchy(ctx, barHier) // can fail if made too quickly
```

Comment:
Hmm, seems the test is still flaky. Was the error you got is something like "the object is updated, please update on a new version"?

/hold